tasks/Word completion given starting letter.md

```diff
@@ -0,0 +1,49 @@
+# Summary
+Groups had to come up with as many words given a starting letter and ending letter as they could in a limited period of time, either individually or in groups of 2, 3, 4, or 5.
+
+# References
+Main paper: https://www-jstor-org.proxy.library.upenn.edu/stable/pdf/40931726.pdf?refreqid=excelsior%3A2597c5c409a1a44999fc96512f1dc106
+> Supplementary: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM_Revision_1.pdf
+
+# Stimuli
+## The visual components
+None listed
+
+# Procedure
+## Steps
+### Pre-survey
+> Individuals completed the Wonderlic Personnel Test (WPT), a 50-item measure that is widely used for assessing individual intelligence.
+> 
+> Individuals completed the the NEO Five Factor Inventory. 
+> 
+> Individuals completed a RME (Readoing the Mind in Eyes) test
+### Task
+> Participants were randomly assigned to work by themselves or in a team of 2, 3, 4, or 5 members
+> 
+> Groups were given 10 minutes to come up with as many words beginning with "s" and ending with "n" as possible.
+> Groups earned 1 point for each nonredundant English word
+### Post-survey
+> Group satisfaction was measured; Participants indicated their agreement to statements such as “I have been very satisfied working on this team” on a five-point scale.
+> 
+> Motivation was measured; Participants indicated their agreement to statements such as “I would feel bad and unhappy if our team has performed poorly” (reverse-scored) on a five-point scale.
+> 
+> Social cohesiveness was measured: Participants indicated their agreement to statements such as “Members of this group would enjoy being at a party together” on a five-point scale.
+>  
+> Psychological safety was measured; Participants indicated their agreement to statements such as “It is difficult to
+ask other members of this team for help” on a five-point scale.
+
+## Roles 
+Groups of 1-5 members all shared the same role
+
+## Instructions
+1. Participants were instructed to individually complete personality tests
+2. Participants were given 10 minutes to come up with as many words beginning with "s" and ending with "n" as possible
+3. Participants were instructed to complete the post-survey scales
```

Comment:
Should these be written as instructions? 

I normally do

Instructions written by CSS lab and would put something like: 
Instructions developed by CSS lab 

You will have 10 minutes to come up with as many words as you can that start with S and end with N 
Your group will earn 1 point for each correctly spelled English word you complete.